ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -158,9 +159,16 @@ public void execute(
     } catch (final Exception e) {
       throw new KsqlException(createInsertFailedExceptionMessage(insertValues), e);
     }
+
+    final String message = String.format(
+        "Inserted:%nkey:%s%nvalue:%s%n",
+        producerRecordInfo.row.key.get("ROWKEY"),
+        producerRecordInfo.row.value
+        );
+    return Optional.of(message);
```

Comment:
nit: can you move the return to within the catch block please?

Also, why not have the message formatting in the `ProducerRecordInfo` class?

```java
   try {
      producer.sendRecord(producerRecordInfo.record,
          serviceContext, config.getProducerClientConfigProps());

      return Optional.of(producerRecordInfo.toString());
   } ...
```

Reply:
ok